UI/admin.html

```diff
@@ -8,255 +8,101 @@
+                                <option>Monday kyeyune</option>
+                                <option>Nambi Rita</option>
+                            </select>&nbsp;<input type="submit" value="Add" id="sendinmail" onclick="myFunc()">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><textarea></textarea></td>
+                    </tr>
+                </tbody>
+                <tfoot>
+                    <tr>
+                        <td><input type="submit" value="Save Group" id="sendinmail" onclick="myFunc()"></form>
+                        </td>
+                    </tr>
+                </tfoot>
             </table>
         </div>
         <div id="Groups" class="tabcontent">
             <table>
-
-
-                    <tr><th><h2>Add Users to groups</h2>
+                <tr>
+                    <th>
+                        <h2>Add Users to groups</h2>
                     </th>
                 </tr>
-                    <tbody>
-                     <tr>
-                         <td><form name="form" onsubmit="required()">
-                               
-                                        <label>Active Groups: </label> &nbsp;
-                                        <select name="textarea" height="4" width="10">
-                                            <option>Select</option>
-                                            <option>Cohort 1</option>
-                                            <option>Cohort 2</option>
-                                            <option>Cohort 3</option>
-                                            <option>Cohort 4</option>
-                                        </select>&nbsp;
-                                            <label>Add contact: </label> &nbsp;
-                                            <select name="textarea" height="4" width="10">
-                                                <option>Select</option>
-                                                <option>Ivan kivumbi</option>
-                                                <option>Ahmed kyakulubya</option>
-                                                <option>Monday kyeyune</option>
-                                                <option>Nambi Rita</option>
-                                            </select>&nbsp;<input type="submit" value="Add" id="sendinmail" onclick="myFunc()">
-                                           <br><br><br><textarea></textarea></td></tr>
-                                       </tbody>
-                               <tfoot>
-                                       <tr><td><input type="submit" value="Save Group" id="sendinmail" onclick="myFunc()"></form></td></tr>
-                               </tfoot>
-
+                <tbody>
+                    <tr>
+                        <td>
+                            <form name="form" onsubmit="required()">
+
```

Comment:
Please remove this blank line

Reply:
Thank you @mendozabree I am on it.

Reply:
@mendozabree I have pushed the new changes. Kindly please review the changes